src/app/components/FormInput/FormInput.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import styled from 'styled-components'
+
+type FormInputProps = {
+  type: 'text' | 'textArea'
+  value: string
+  name: string
+  diff?: number
```

Comment:
It took me a while to figure out what `diff` is, maybe think about renaming it? 
What about `charactersLeft` for example?